.gitignore

```diff
@@ -5,4 +5,4 @@ request.log
 response.log
 .logs/
 .logs
-manifest.yaml
+// manifest.yaml
```

Comment:
Huh, that was not really intended I think/is a leftover from first setting this up. 

Please just remove the entry completely, it should not be here. 

Thanks for catching that! :)